src/sbt-test/sbt-source-dist/simple-with-name/build.sbt

```diff
@@ -0,0 +1,4 @@
+scalaVersion              := "2.13.10"
+version                   := "0.1.9"
+name                      := "incubator-pekko"
```

Comment:
This test is checking the specific case where we automatically derive `sourceDistName` just from `name`